41951 Checkbox "Show property status" in unit lists

dai-eastgate opened the pull request:
related to https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/642
changed log:
Update the checkbox to show estate status in unit lists

dai-eastgate committed:
41951 update checkbox show estate status for unit lists

dai-eastgate committed:
41951 update code

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

andernath commented on plugin/Model/FormModelBuilder/FormModelBuilderDBEstateUnitListSettings.php:
```diff
@@ -234,4 +235,20 @@ public function getInputModelCustomLabelLanguageSwitch(): InputModelDB
 		});
 		return $pInputModel;
 	}
+
```
This function already exist in FormModelBuilderDBEstateListSettings
Why would you duplicate it here?

dai-eastgate replied:
Sorry for the confusion. The function was indeed duplicated, and I have updated the code to remove it. Could you please review it again? Thank you.

andernath replied:
No problem @dai-eastgate :)
Could you first take a look at the unit tests please?

dai-eastgate replied:
@andernath The unit tests have been addressed in the PR at [oo-wp-plugin#792](https://github.com/onOffice-Web-Org/oo-wp-plugin/pull/792). Could you please review it first?

andernath replied:
@dai-eastgate Ops sorry I my fault, missed your comment from yesterday. Of course I will start reviewing fist. :)

andernath reviewed (commented):
@dai-eastgate I have a question. Please take a look.

dai-eastgate committed:
41951 refactor code

dai-eastgate committed:
41951 update unit test

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate commented:
@fredericalpers  I will check it

dai-eastgate committed:
41951 add condition show property status for unit detail

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate commented:
@fredericalpers @andernath I fixed it and during testing, I discovered an issue where the unit detail was not loading the necessary script. I've fixed it in this PR, could you take a look?
Here's a video demo: https://files.fm/u/xd5qm6bezg#/view/4gxenfcyxk

dai-eastgate committed:
41951 refactor code

andernath committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

fredericalpers commented:
@dai-eastgate unfortunately I think I did not make it clear enough. As the property status gets inherited from the "normal" detail page, even tho I disabled it for unit detail pages, it will still show.

Show property status disabled for unit detail pages = no property status should be displayed on unit detail pages at all. Currently still the inherited one from "normal" detail pages will be displayed.

dai-eastgate committed:
41951 update condition show status in unit list

dai-eastgate commented:
> @dai-eastgate unfortunately I think I did not make it clear enough. As the property status gets inherited from the "normal" detail page, even tho I disabled it for unit detail pages, it will still show.
> 
> Show property status disabled for unit detail pages = no property status should be displayed on unit detail pages at all. Currently still the inherited one from "normal" detail pages will be displayed.

@fredericalpers @andernath  I have fixed it and here is video demo, please review and test this branch again. Thanks!

https://files.fm/u/ey5z2d86xd#/view/xhb9zy26bf

dai-eastgate committed:
41951 remove load script unit

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

yeneastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
Merge branch 'master' into 41951-checkbox-show-property-status-in-unit-lists

dai-eastgate committed:
41951 refactor code
